govet.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+set -ex
+
+go vet ./... -mod=mod
```

Comment:
maybe change order to `go vet -mod=mod ./...` ? I get this locally:
```
go vet ./... -mod=mod
malformed import path "-mod=mod": leading dash
```

Reply:
trying now